cli/tests/integrations/test_package.py

```diff
@@ -179,24 +232,40 @@ def test_describe_app_cli():
 def test_describe_specific_version():
     stdout = file_bytes(
         'tests/data/package/json/test_describe_marathon_0.11.1.json')
-    assert_command(['dcos', 'package', 'describe', '--package-version=0.11.1',
-                    'marathon'],
-                   stdout=stdout)
+
+    returncode_, stdout_, stderr_ = exec_command(
+        ['dcos', 'package', 'describe', '--package-version=0.11.1',
+         'marathon'])
+
+    assert returncode_ == 0
+    output = json.loads(stdout_.decode('utf-8'))
+    assert _remove_nulls(output) == json.loads(stdout.decode('utf-8'))
+    assert stderr_ == b''
 
 
 def test_bad_install():
     args = ['--options=tests/data/package/chronos-bad.json', '--yes']
-    stderr = b"""Error: False is not of type 'string'
+    stdout = b""
+    if os.environ.get(constants.COSMOS_URL_ENV) is None:
+        stderr = b"""Error: False is not of type 'string'
 Path: chronos.zk-hosts
 Value: false
 
 Please create a JSON file with the appropriate options, and pass the \
 /path/to/file as an --options argument.
 """
+    else:
+        stderr = b"""Error: Options JSON failed validation
+Found: boolean
+Expected: string
+Path: chronos.zk-hosts
 
+Please create a JSON file with the appropriate options, and pass the \
+/path/to/file as an --options argument.
+"""
```

Comment:
The error is different because we don't know the actual value the user passed it from the cosmos error. Do we want to change cosmos to give us this information, or find in the cli?